app/assets/javascripts/client/controllers/learning_questions/LearningQuestionListDisplayCtrl.js

```diff
@@ -0,0 +1,61 @@
+(function() {
+  'use strict';
+  angular.module('PDRClient')
+      .controller('LearningQuestionListDisplayCtrl', LearningQuestionListDisplayCtrl);
+
+  LearningQuestionListDisplayCtrl.$inject = [
+      '$stateParams',
+      '$scope',
+      '$window',
+      'LearningQuestion'
+  ];
+
+  function LearningQuestionListDisplayCtrl($stateParams, $scope, $window, LearningQuestion) {
+    var vm = this;
+    vm.learningQuestions = [];
+
+    vm.extractId = function() {
+      return $stateParams.assessment_id || $stateParams.id;
+    };
+
+    vm.loadQuestions = function () {
+      LearningQuestion
+          .get({assessment_id: vm.extractId()})
+          .$promise
+          .then(function (result) {
+            vm.learningQuestions = result.learning_questions;
+          });
+    };
+
+    vm.deleteLearningQuestion = function (model) {
+      var willDelete = $window.confirm('Are you sure you wish to delete this learning question?');
+      if (willDelete) {
+        LearningQuestion
+            .delete({assessment_id: $stateParams.id, id: model.id})
+            .$promise
+            .then(function () {
+              $scope.$emit('learning-question-change');
+            });
+      }
+    };
+
+    vm.updateLearningQuestion = function (model) {
+      if (model.editable) {
```

Comment:
My comments earlier on early returns apply, but this is also a sanity guard.  We don't show the edit button or (if through DOM manipulation they managed to invoke it) allow the user to edit the content of the question, either.

This did highlight a case I didn't catch with the delete above; we also need to be sure that the entity is editable before we even show the confirm dialog.